includes/class-cron.php

```
@@ -136,20 +136,23 @@ private function schedule_page_cron_jobs(): void {
 			return;
 		}
 
-		$options      = get_option( 'wppo_settings', array() );
-		$exclude_urls = Util::process_urls( $options['preload_settings']['excludePreloadCache'] ?? array() );
+		$options          = get_option( 'wppo_settings', array() );
+		$raw_exclude_urls = Util::process_urls( $options['preload_settings']['excludePreloadCache'] ?? array() );
+
+		$normalized_exclude_urls = array();
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $normalized_exclude_urls. Keep variable name length under 20. (undefined)

(LongVariable)